docs/developer-guide/rules.md

```diff
@@ -232,42 +148,36 @@ The final step is to add references to the new rule in the following places:
 -   [The rules `index.js` file](../../lib/rules/index.js)
 -   [The list of rules](../user-guide/rules/list.md)
 
-Once you have something to show, you'll create a [pull request](https://github.com/stylelint/stylelint/compare) to continue the conversation.
-
-## Adding an option to an existing rule
+## Add an option to a rule
 
-First, open [an issue](https://github.com/stylelint/stylelint/issues/new) about the option you wish to add. We'll discuss its functionality and name there.
-
-Once we've agreed on the direction, you can work on a pull request. Here are the steps you'll need to take:
+You should:
 
 1.  Run `npm run watch` to start the interactive testing prompt.
 2.  Use the `p` command to filter the active tests to just the rule you're working on.
 3.  Change the rule's validation to allow for the new option.
-4.  Add to the rule some logic (as little as possible) to make the option work.
-5.  Add new unit tests to test the option.
+4.  Add new unit tests to test the option.
+5.  Add (as little as possible) logic to the rule to make the tests pass.
 6.  Add documentation about the new option.
```

Comment:
[question] Should we start with a lower case character as follows?

```
You should:

1.  run ...
2.  use ....
3.  change ...
...
```

Reply:
I tried to consistently follow the [Gov UK standard](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#bullet-points-steps) of:

-   _bullets_ with lower case characters and no periods
-   _steps_ in sentence case with periods

I figured we should stick with it as it's a well-respected guide.

Reply:
Thanks for your explanation, I understand! 👍